runtime/onert/backend/train/BackendContext.cc

```diff
@@ -48,16 +48,16 @@ void BackendContext::genDerivativeTensors()
   auto tensor_builder = _deriv_tensor_builder;
   auto tensor_reg = _deriv_tensor_registry;
 
-  tgraph.operands().iterate([&](const ir::OperandIndex &ind, const ir::Operand &) {
+  tgraph.operands().iterate([&](const ir::OperandIndex &ind, const ir::Operand &obj) {
     if (external_operands().contains(ind))
       return;
     // NOTE Assuming there is no layout changes (Always assume NHWC or UNKNOWN)
     assert(tgraph.layout() != ir::Layout::NCHW);
 
-    // TODO Register TensorInfo that has the shape of differentiation and differentiation
-    // ir::OperandInfo backend_info{obj.shape(), obj.typeInfo(), obj.info().memAllocType(),
-    //                              obj.isConstant()};
-    // tensor_builder->registerTensorInfo(ind, backend_info, ir::Layout::NHWC);
+    // TODO Different shape of deriv tensor
+    ir::OperandInfo backend_info{obj.shape(), obj.typeInfo(), obj.info().memAllocType(),
+                                 obj.isConstant()};
+    tensor_builder->registerTensorInfo(ind, backend_info, ir::Layout::NHWC);
```

Comment:
You can register `TensorInfo` obtained from the function below. But it is still empty.

https://github.com/Samsung/ONE/blob/3306f0483ca8b579a2072bf30bd0fcbfa6850631/runtime/onert/core/include/ir/train/TrainableGraph.h#L121C2-L121C2